command/v7/create_space_quota_command.go

```diff
@@ -9,16 +9,16 @@ import (
 type CreateSpaceQuotaCommand struct {
 	BaseCommand
 
-	RequiredArgs          flag.SpaceQuota          `positional-args:"yes"`
-	NumAppInstances       flag.IntegerLimit        `short:"a" description:"Total number of application instances. (Default: unlimited)."`
-	PaidServicePlans      bool                     `long:"allow-paid-service-plans" description:"Allow provisioning instances of paid service plans. (Default: disallowed)."`
-	PerProcessMemory      flag.MemoryWithUnlimited `short:"i" description:"Maximum amount of memory a process can have (e.g. 1024M, 1G, 10G). (Default: unlimited)."`
-	TotalMemory           flag.MemoryWithUnlimited `short:"m" description:"Total amount of memory all processes can have (e.g. 1024M, 1G, 10G). -1 represents an unlimited amount. (Default: 0)."`
-	TotalRoutes           flag.IntegerLimit        `short:"r" description:"Total number of routes. -1 represents an unlimited amount. (Default: 0)."`
-	TotalReservedPorts    flag.IntegerLimit        `long:"reserved-route-ports" description:"Maximum number of routes that may be created with ports. -1 represents an unlimited amount. (Default: 0)."`
-	TotalServiceInstances flag.IntegerLimit        `short:"s" description:"Total number of service instances. -1 represents an unlimited amount. (Default: 0)."`
-	usage                 interface{}              `usage:"CF_NAME create-space-quota QUOTA [-m TOTAL_MEMORY] [-i INSTANCE_MEMORY] [-r ROUTES] [-s SERVICE_INSTANCES] [-a APP_INSTANCES] [--allow-paid-service-plans] [--reserved-route-ports RESERVED_ROUTE_PORTS]"`
-	relatedCommands       interface{}              `related_commands:"create-space, space-quotas, set-space-quota"`
+	RequiredArgs          flag.SpaceQuota             `positional-args:"yes"`
+	NumAppInstances       flag.IntegerLimit           `short:"a" description:"Total number of application instances. (Default: unlimited)."`
+	PaidServicePlans      bool                        `long:"allow-paid-service-plans" description:"Allow provisioning instances of paid service plans. (Default: disallowed)."`
+	PerProcessMemory      flag.MegabytesWithUnlimited `short:"i" description:"Maximum amount of memory a process can have (e.g. 1024M, 1G, 10G). (Default: unlimited)."`
```

Comment:
@pivotalgeorge I think we separated the UI description from the function of the flag in our mind. We thought that `MegabytesWithUnlimited` made sense as the flag name because the output value of that flag will always be in megabytes. Versus our new flag, where the output value is always in bytes.

Reply:
gotcha. On that point--the new flag being always-bytes--is there a good reason for this? It feels like keeping units consistent would be desirable

Reply:
Yes, that's right!